src/libPOW/pow.cpp

```diff
@@ -157,7 +166,8 @@ bool POW::EthashConfigureClient(uint64_t block_number, bool fullDataset) {
     m_epochContextLight = ethash::create_epoch_context(epochNumber);
   }
 
-  bool isMineFullCpu = fullDataset && !CUDA_GPU_MINE && !OPENCL_GPU_MINE;
+  bool isMineFullCpu =
+      fullDataset && !CUDA_GPU_MINE && !OPENCL_GPU_MINE && !GETWORK_SERVER_MINE;
```

Comment:
Change to bool isMineFullCpu =
      fullDataset && !CUDA_GPU_MINE && !OPENCL_GPU_MINE && !GETWORK_SERVER_MINE && !REMOTE_MINE;

Reply:
got it.